spring-integration-jdbc/src/main/sql/destroy.sql.vpp

```diff
@@ -12,7 +12,7 @@ DROP TABLE $!{IFEXISTSBEFORE} INT_MESSAGE_GROUP $!{IFEXISTS};
 DROP TABLE $!{IFEXISTSBEFORE} INT_GROUP_TO_MESSAGE $!{IFEXISTS};
 DROP TABLE $!{IFEXISTSBEFORE} INT_LOCK $!{IFEXISTS};
 DROP TABLE $!{IFEXISTSBEFORE} INT_CHANNEL_MESSAGE $!{IFEXISTS};
+DROP TABLE $!{IFEXISTSBEFORE} INT_METADATA_STORE $!{IFEXISTS};
 #if(${INT_MESSAGE_SEQ})
 DROP SEQUENCE INT_MESSAGE_SEQ $!{IFEXISTS};
-#end
-
+#end
```

Comment:
Please, consider to switch on the option in your IDE to add new line in the end of files on save.